### PR TITLE
 Fix 1031: Onboarding Flow landscape 

### PR DESCRIPTION
**Pull request by nikitamarysolomanpvt**
## Explanation
Duplicate of #1032 created for branch name change.
Please refer https://github.com/oppia/oppia-android/pull/1032 for addressed comments.
Onboarding Flow landscape
## Mocks
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/1f86ca4e-795e-4a75-8784-fea53864b94b/RG-OB-Onboarding-1-
The following is a list of opportunities to improve the accessibility of Oppia. Each item corresponds to an outlined area on the attached screenshot.

##GIF 
mobizen_20200429_150210 (1)

## Accessibility Scanner Results:
Item label
org.oppia.app:id/onboarding_fragment_next_image_view
This item may not have a label readable by screen readers.

Item descriptions
org.oppia.app:id/slide_image_view
This non-clickable item's speakable text: "Welcome to Oppia!" is identical to that of 1 other item(s).
screenshot_Oppia_2020-04-29-15_08_10

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/onboarding_slide_final.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      android:textSize="16sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.0"
+      app:layout_constraintStart_toEndOf="@+id/slide_image_view"
+      app:layout_constraintTop_toBottomOf="@+id/slide_title_text_view" />
+
+    <Button
+      android:id="@+id/get_started_button"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
```
Comment (rt4914) [inline app/src/main/res/layout-land/onboarding_slide_final.xml:66]: The earlier comment was misleading from my side: https://github.com/oppia/oppia-android/pull/1032#discussion_r417210478

The marginStart is `28dp`

File: app/src/main/res/layout-land/onboarding_slide_final.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.onboarding.OnboardingSlideViewModel" />
```
Comment (rt4914) [inline app/src/main/res/layout-land/onboarding_slide_final.xml]: This view model is incorrect.

Comment (rt4914) [reply app/src/main/res/layout-land/onboarding_slide_final.xml]: In portrait all these details are set from strings.xml file directly but in this you are trying to use viewmodel (which is not necessary). The viewmodel should be used consistently in landscape and portrait layout.

**Review by rt4914 — CHANGES_REQUESTED** [review]
There are issues in this PR, I have the left the comment, you can simple try to run this branch and check for errors.

**Comment by rt4914** [comment]
@nikitamarysolomanpvt Also, I saw your that it was assigned to me that's why I reviewed it but now I see that it was unassigned later on.

File: app/src/main/res/layout-land/onboarding_slide_final.xml (hunk 2 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/res/layout-land/onboarding_slide_final.xml]: @rt4914 I had unassigned you as i was supposed to make changes once i complete will assign you back.

File: app/src/main/res/layout-land/onboarding_slide_final.xml (hunk 1 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/res/layout-land/onboarding_slide_final.xml:66]: Done

File: app/src/main/res/layout-land/onboarding_slide_final.xml (hunk 2 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/res/layout-land/onboarding_slide_final.xml]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by veena14cs — APPROVED** [review]
LGTM.

**Pull request merged**